### DISCUSSION TITLE
Ayuda con el mensaje de bienvenida |cffFF4A2DThis server runs on AzerothCore|r |cff3CE7FFwww.azerothcore.org|r como puedo cambiarlo.

### DISCUSSION THREAD

**Discussion by taylon20** [azerothcore/azerothcore-wotlk#17206] [General]
Buenas a todos alguien me podría decir como cambiar el mensaje de bienvenida 
|cffFF4A2DThis server runs on AzerothCore|r |cff3CE7FFwww.azerothcore.org|r

**Comment by Nyeriah** [comment]
You shouldn't remove the credits message.

**Comment by taylon20** [comment]
Solo pregunto por conocimiento ya que he estado revisando yo y unos colegas y no encontramos y hemos buscado información al respecto y no hemos encontrado. Serian tan amables de decirnos como
